testing/trino-server-dev/etc/config.properties

```diff
@@ -55,4 +55,5 @@ plugin.bundles=\
   ../../plugin/trino-exchange-hdfs/pom.xml, \
   ../../plugin/trino-mysql-event-listener/pom.xml
 
+plugin.dir=etc/plugin
```

Comment:
can this also be an absolute path? That would be good .. then you could have a plugin in a separate Maven project and point at its target folder or similar. In either case .. we should document this.

Reply:
Absolute is fine, yes. I clarified in a comment.

Reply:
Where .. I guess you forgot to push..

Reply:
oops :) typed `git push` and didn't hit enter I guess ... actually pushed now!